app/components/UI/Perps/components/PerpsOrderHeader/PerpsOrderHeader.tsx

```diff
@@ -105,7 +105,7 @@ const PerpsOrderHeader: React.FC<PerpsOrderHeaderProps> = ({
       </View>
       <TouchableOpacity onPress={handleOrderTypePress}>
         <View style={styles.marketButton}>
-          <Text variant={TextVariant.BodyMD} color={TextColor.Alternative}>
+          <Text variant={TextVariant.BodyMD} color={TextColor.Default}>
```

Comment:
Nit: Don't think we need to explicitly specify the default color.
```suggestion
          <Text variant={TextVariant.BodyMD}>
```